cf/cognito-case-insensitive.yaml

```diff
@@ -199,11 +199,10 @@ Resources:
       UsernameConfiguration:
         CaseSensitive: false
       LambdaConfig:
-        PostConfirmation: 
-          Fn::ImportValue: !Sub PostRegisterLambdaArn-${Environment}
+        PostConfirmation:
+          Fn::ImportValue: !Sub PostRegistrationLambdaArnV2-${Environment}
```

Comment:
So now the lambda does the user invite for both v1 and v2? If thats true then you should maybe delete one of the post lambda yaml files because its not used?

Reply:
Yup. Oh yea, I'll delete the old lambda script